ng-dev/format/cli.ts

```diff
@@ -46,11 +46,19 @@ export function buildFormatParser(localYargs: yargs.Argv) {
     .command(
       'staged',
       'Run the formatter on all staged files',
-      (args) => args,
-      async ({check}) => {
+      (args) =>
+        args.options('add', {
+          type: 'boolean',
+          description: 'Add the potentially formatted files back to the staging area',
+          conflicts: 'check',
+        }),
+      async ({check, add}) => {
         const executionCmd = check ? checkFiles : formatFiles;
         const allStagedFiles = GitClient.get().allStagedFiles();
         process.exitCode = await executionCmd(allStagedFiles);
+        if (!check && add && process.exitCode === 0) {
```

Comment:
@devversion what do you think of just always re-staging the files when `format staged` is executed? We would not run it during a check obviously.

```suggestion
        if (!check && process.exitCode === 0) {
```

Reply:
I've applied the change since I was anyways having issues passing the `--add` flag to the command in the bash pre-commit script (strangely enough it worked perfectly if called directly in the terminal, also `--check` worked just fine in the script :man_shrugging:)

if we don't want it I can always revert the change anyways :slightly_smiling_face: (and keep digging as to why `--add` wouldn't be recognized :sweat_smile: )

Anyways yeah I do agree that it could make sense to have this behavior, I am just wondering, is there any case in which this could be undesirable? (keeping in mind that we do still have the `--check` anyways)

Reply:
@josephperrott Sorry, didn't realize I got mentioned here again. I'm good with always staging (when not checking).